libncpoco/cp_win32.c

```diff
@@ -28,6 +29,72 @@ static const char* driveletters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRS
 /* Forward */
 static int isAbsolutePath(const char* path);
 
+#if !defined(nulldup) && !defined(HAVE_STRDUP)
```

Comment:
Not sure what is the point of this change. Also, I believe we have tried to
avoid use of inline because older versions of some C compilers do not support it.

Reply:
The point of this change is defining a fallback definition of `nulldup` in case `strdup` is not provided by the C runtime.  I don't remember if that was a problem on Cygwin or MinGW, but the current definitions all assume that `strdup` will be provided if `nulldup` isn't.

Reply:
I tried adding `inline`, but since it wasn't in the declaration (or the other twenty definitions) the C compiler rejected it and I took it out of the definition.

Reply:
Good point about strdup; you might consider putting the body of the
alternate nulldup into a more general location such as libdispatch/dutil.c
or putting it into ncconfigure.h as a static function.

Reply:
Moved to `include/ncconfigure.h`